ui/uistateadapter/src/main/java/com/example/uistateadapter/Utils.kt

```
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.uistateadapter
+
+import com.google.jetpackcamera.ui.uistate.SingleSelectableUiState
+
+object Utils {
+
+    /** Given a set of values and a filtering list, return a list that only contains the values that
+     *  are selectable. */
+    fun <T> getSelectableListFromValues(
+        valueSet: Set<T>,
```

is `filteringList` implied to always contain `valueSet`? If so, should we throw an error if a value in `valueSet` is not in `filteringList`?